src/webview/create-component/pages/fromExistingGitRepo.tsx

```diff
@@ -229,7 +250,27 @@ export function FromExistingGitRepo({ setCurrentView }) {
                                                 spacing={2}
                                                 alignItems="center"
                                             >
-                                                <CircularProgress />
+                                                <Box sx={{ position: 'relative', display: 'inline-flex' }}>
+                                                    <CircularProgress />
```

Comment:
It looks like there is a pre-built component for this: https://mui.com/material-ui/react-progress/#circular-with-label, I think it would be better to use the pre-built component

Reply:
I think there were no pre-build component, internally they were used [CircularProgress](https://mui.com/material-ui/react-progress/#circular). I haven't find any specific imports as well

Reply:
You're right. Wow, the MUI documentation is pretty misleading.